model/inp/ww3_shel.inp

```diff
@@ -310,7 +310,7 @@ $ Type 4 : Restart files (no additional data required).
 $                               The data file is restartN.ww3, see
 $                               w3iors.ftn for additional doc.
 $
```

Comment:
Might be useful to extend comment to describe what the two logical flags do, and that they are optional unless set True

Reply:
Thanks Andy, some notes:

2. Adding winds to the restart following the same approach as done in this branch should be straightforward - adding code to include winds in the EXTRA namelist processing, and enclosing all the WRST switch lines with if sentences depending on the appropriate flag for coupling restart.

3. I am not aware there is a ticket for updating the OASIS coupling documentation, although I would start working on it only when the rest of the coupling tickets have been merged.

I will add the modifications you suggest to the code comments.

Reply:
I also added yesterday a small change to remove one warning when the list of output fields is not specified (so it has an 'unset' value), as this warning is supposed to be there to identify fields in the list that are not valid.

Reply:
The changes look good @ukmo-juan-castillo , thanks for these